tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunnerUtils.java

```diff
@@ -0,0 +1,263 @@
+    private static int absLineNum;
+    public static String detailsOfTests = "";
+    public static int totalTest = 0;
+    public static int totalTestPassed = 0;
+
+
+    public static void readTestFile(String fileName, String path, List<Object[]> testCases, Set<String> labels)
+                                                                                                    throws IOException {
+        File testFile = new File(path);
+        FileReader fileReader = new FileReader(testFile);
+        BufferedReader buffReader = new BufferedReader(fileReader);
+        String line = buffReader.readLine();
+        absLineNum = 1;
+        String pattern = String.format("^(%s)\\s*:\\s*.*", START_TEST_CASE);
+        while (line != null) {
+            if (Pattern.matches(pattern, line)) {
+                line = createTestFile(fileName, line, buffReader, testCases, labels);
+            } else {
+                line = buffReader.readLine();
+                absLineNum++;
+            }
+        }
+        buffReader.close();
+    }
+
+    private static String createTestFile(String fileName, String kindOfTestFile, BufferedReader buffReader,
+                                         List<Object[]>  testCases, Set<String> labels) throws IOException {
+        String tempFileName = fileName.substring(0, fileName.indexOf(".")) + fileCount++;
+        String tempDir = TEMP_DIR + tempFileName + BAL_EXTENSION;
+        File tempBalFile = new File(RESOURCE_DIR + tempDir);
+        String kindOfTestCase = readHeadersOfTestCase(buffReader, kindOfTestFile, labels);
+        if (kindOfTestCase.equals(SKIPPED_TEST)) {
+            return EMPTY_STRING;
+        }
+        List<String> outputValues = new ArrayList<>();
+        List<Integer> errLines = new ArrayList<>();
+        Object[] testCase = new Object[6];
+        testCase[0] = kindOfTestCase;
+        FileWriter tempFileWriter = new FileWriter(tempBalFile);
+        boolean isOutputTest = kindOfTestCase.equals(OUTPUT);
+        if (isOutputTest) {
+            tempFileWriter.write(IMPORT_BALLERINAI + NEW_LINE_CHARACTER);
+        }
+        String line;
+        int relativeLineNum = 0;
+        while ((line = buffReader.readLine()) != null) {
+            if (!isOutputTest) {
+                relativeLineNum = relativeLineNum + 1;
+            }
+            if (line.startsWith(START_TEST_CASE)) {
+                break;
+            }
+            String pattern = String.format(".*//\\s*@\\s*(%s)\\s*.*", kindOfTestCase);
+            if (Pattern.matches(pattern, line)) {
+                String output = line.substring(line.indexOf(kindOfTestCase) + kindOfTestCase.length()).trim();
+                if (!kindOfTestCase.equals(OUTPUT)) {
+                    errLines.add(relativeLineNum);
+                }
+                outputValues.add(output);
+            } else if (Pattern.matches(".*//.*", line)) {
+                Assert.fail("format of output is incorrect, format of output " +
```

Comment:
```
Test-Case: output
Description: Test 2 for integer multiplication overflow, where one operand is of type optional int:Unsigned8.
Labels: multiplicative-expr, int, int-literal

public function main() {
    int a = -461168601842738791;
    int b = 20;
    int c = a * b; // @panic integer overflow on multiplication
}
```

Here, the kind and the comment have a mismatch, and results in this error.

```
[Utils] [ERROR] [Error] java.lang.AssertionError: format of output is incorrect, format of output should be //@error, //@output and //@panic
```

IMO, we need to be more specific and maybe say something like

```
invalid test comment '@panic' for test kind 'output'; expected '@output'
```